tests/jwk/test_jwk_set.py

```diff
@@ -76,3 +76,18 @@ def test_key_set_iter(self):
         key_set = KeySet.generate_key_set('RSA', 2048)
         for k in key_set:
             self.assertEqual(k.key_type, "RSA")
+
+    def test_key_eq_with_same_keys(self):
+        key_set1 = KeySet.generate_key_set('RSA', 2048)
```

Comment:
Nitpick but generating all these key/sets repeatedly in every test is inefficient in terms of wall/CPU time and energy.

Consider using [setUpClass()](https://github.com/authlib/joserfc/pull/37/files#diff-a26577d21b8774c2dbb2fafa20ffbbb67abce1d96f1ba31a551e45412bef11b0R13) or [module-scoped Pytest fixtures](https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#scope-sharing-fixtures-across-classes-modules-packages-or-session] to make the tests run faster / be kinder to your (+ everyone else's) computer and the planet.

Reply:
@bjmc Thanks for the suggestion. I've reduced the usage of generate_key in eb5f55cd68243a5afee94077a272de317b43a650